Gemfile

```
@@ -3,9 +3,9 @@ source "https://rubygems.org"
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby ">= 2.6.10"
 
-gem "cocoapods", "~> 1.13"
+gem "cocoapods", "~> 1.14", ">= 1.14.0"
 gem "activesupport", ">= 6.1.7.3", "< 7.1.0"
-gem "fastlane", "~> 2"
+gem "fastlane", "~> 2", ">= 2.220.0"
```

```suggestion
gem "fastlane", "~> 2"
```

I would not change it how it's written, I would just run `bundle update fastlane` and commit the changes to Gemfile.lock

---

I'll do the same for cocoapods 👀

---

Thank you, we should update that one to `"~> 1.14"`